app/src/main/kotlin/uk/govuk/app/ui/GovUkApp.kt

```diff
@@ -241,50 +244,55 @@ private fun BottomNavScaffold(
 private fun homeScreenWidgets(
     navController: NavHostController,
     isSearchEnabled: Boolean,
+    isRecentActivityEnabled: Boolean,
     isTopicsEnabled: Boolean,
     onClick: (String) -> Unit
 ): List<@Composable (Modifier) -> Unit> {
-    return listOf(
-        { modifier ->
-            if (isSearchEnabled) {
-                SearchWidget(
-                    onClick = { text ->
-                        onClick(text)
-                        navController.navigate(SEARCH_GRAPH_ROUTE)
-                    },
-                    modifier = modifier
-                )
-            }
-        },
-        { modifier ->
+    val widgets = mutableListOf<@Composable (Modifier) -> Unit>()
```

Comment:
Had to rework creating this list as previously elements were still added to the list regardless of wether the widget is enabled or not causing unwanted spacing to be added.

Reply:
Good spot!